CHANGELOG.next.asciidoc

```diff
@@ -947,6 +947,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
+- The experimental modules for Citrix Netscaler and Symantec Endpoint Protection
```

Comment:
The experimental modules for Citrix Netscaler and Symantec Endpoint Protection have been removed. As we continue to expand our coverage of common security data sources, we may consider supporting Citrix Netscaler and Symantec Endpoint Protection in a future release. 

@andrewkroh that sound ok?

Reply:
Incorporated. 👍